api_tests/users/views/test_user_list.py

```diff
@@ -122,27 +122,38 @@ def test_user_can_not_create_other_user_or_send_mail(self, mock_mail):
             expect_errors=True
         )
 
-        assert_equal(res.status_code, 403)
+        assert_equal(res.status_code, 201)
```

Comment:
Hmmm. Probably cookie and appropriately-scoped token are good, maybe no on the basic auth.